x-pack/plugins/profiling/kibana.jsonc

```diff
@@ -6,16 +6,14 @@
     "id": "profiling",
     "server": true,
     "browser": true,
-    "configPath": [
-      "xpack",
-      "profiling"
-    ],
+    "configPath": ["xpack", "profiling"],
     "requiredPlugins": [
       "navigation",
       "data",
       "kibanaUtils",
       "share",
       "observability",
+      "observabilityShared",
```

Comment:
What's the difference between `observability` and `observability_shared`? By reading each README file looks like they exist for the same purpose.

Reply:
@cauemarcondes 

There's two primary reasons for this change, one conceptual and one practical.

1. When the Observability app started, it was just the Overview page. It made sense to use it as a sharing place for all downstream Observability apps. Over time a lot more routes have been added: we are now at 8 routes and counting. There is so much dedicated logic inside the app that it starts to make less sense to keep it as a shared repository dictating how the other apps should work, but that it should also be a downstream app that consumes shared resources like all other Observability apps.
2. The Synthetics team has placed a component in the Observability app which requires the Exploratory View app. But the Exploratory View app requires the Observability app as it shared the same navigation. This change fixes this circular dependency issue.

Reply:
Does the `Observability` plugin also depend on the `Observability_shared` plugin? Or can it? Is there a plan to "clean up" the observability plugin by moving stuff that is actually shared to the new plugin?

Reply:
@cauemarcondes 

> Does the Observability plugin also depend on the Observability_shared plugin? 

Yes, or at least, it will in an upcoming PR.

> Is there a plan to "clean up" the observability plugin by moving stuff that is actually shared to the new plugin?

Yes, I intend to kick out all shared stuff to Observability_shared.